src/utils/setup/create-video-pages.ts

```diff
@@ -0,0 +1,21 @@
+import path from 'path';
+import { fetchBuildTimeMedia } from './fetch-build-time-media';
+import { Video } from '../../interfaces/video';
+
+export const createVideoPages = async (
+    createPage: Function,
+    metadataDocument: object
```

Comment:
```suggestion
    metadata: object
```

If we rename here then on line 16 we can use shorthand to be `metadata` only.